### PR TITLE
[BAHIR-101] updates to sql-cloudant readme and python examples

### PR DESCRIPTION
**Pull request by emlaver**
- Updated documentation according to the README.md template so it can be compiled into http://bahir.apache.org/docs/spark/current/documentation/ automatically
- Fixed print call when testing examples in Python 3.x

@yanglei99 FYI

### CODE REVIEW

**Comment by ApacheBahir** [comment]
Can one of the admins verify this patch?

File: sql-cloudant/README.md [hunk 1]
```diff
@@ -1,24 +1,10 @@
...
-    2. [Using SQL In Scala](#Using-SQL-In-Scala)
-    3. [Using DataFrame In Python](#Using-DataFrame-In-Python)
-    4. [Using DataFrame In Scala](#Using-DataFrame-In-Scala)
-    5. [Using Streams In Scala](#Using-Streams-In-Scala)
-4. [Configuration Overview](#Configuration-Overview)
-5. [Known limitations and areas for improvement](#Known-limitations)
-
-
-<div id='Linking'/>
+[IBM® Cloudant®](https://cloudant.com) is a document-oriented DataBase as a Service (DBaaS). It stores data as documents
```
Comment (lresende) [inline sql-cloudant/README.md]: Can we make sure we also mention that this works for vanilla Apache CounchDB as well.

**Review by lresende — CHANGES_REQUESTED** [review]

**Comment by lresende** [comment]
ok to test

**Comment by ApacheBahir** [comment]
Build successful

**Comment by ApacheBahir** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://169.45.79.58:8080/job/bahir_spark_pr_builder/51/

**Comment by ApacheBahir** [comment]
Build successful

**Comment by ApacheBahir** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://169.45.79.58:8080/job/bahir_spark_pr_builder/52/

**Comment by lresende** [comment]
@emlaver Sorry, I didn't realized this was re-using the same jira for cloudant initial code. Next time create a new jira so it becomes easier to count your contributions.

**Comment by emlaver** [comment]
@lresende Ok, no problem, I'll make sure to open a new JIRA for the next one.

**Pull request closed without merging**